### ISSUE TITLE
"mem is not a function" using si.getAllData()

### ISSUE THREAD

**Issue by ACXgit** [sebhildebrandt/systeminformation#45]
Hello, I get this error when using `si.getAllData()`:

```
node_modules/systeminformation/lib/index.js:267
      mem().then(res => {
      ^

TypeError: mem is not a function
    at process.nextTick (node_modules/systeminformation/lib/index.js:267:7)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

I'm on Arch Linux (4.7.2-1-ARCH #1 SMP PREEMPT Sat Aug 20 23:02:56 CEST 2016 x86_64 GNU/Linux), Node v8.1.2, `"systeminformation": "^3.23.3"` in package.json.

**Comment by sebhildebrandt** [comment]
Will fix it today ... sorry for the inconvenience

**Comment by ACXgit** [comment]
I've just made a pull request with a fix:
https://github.com/sebhildebrandt/systeminformation/pull/46
...and no need to apologize! :) Thank you instead for your effort developing this lib.

**Comment by sebhildebrandt** [comment]
Done! Fixed! New version v3.23.4 just published. Thank you for reporting this issue. Whenever you discover some other issue or have comments, do not hesitate to contact me.
